server/routes/creatingLicences/handlers/caseload.test.ts

```diff
@@ -225,73 +251,13 @@ describe('Route Handlers - Create Licence - Caseload', () => {
     })
 
     it('should successfully search by offender name', async () => {
-      req.query = { view: 'team', search: 'rogan' }
-
-      await handler.GET(req, res)
-      expect(res.render).toHaveBeenCalledWith('pages/create/caseload', {
-        caseload: [
-          {
-            name: 'Joe Rogan',
-            crnNumber: 'X381306',
-            releaseDate: '12 Oct 2022',
-            prisonerNumber: '123',
-            licenceStatus: LicenceStatus.IN_PROGRESS,
-            licenceType: LicenceType.AP,
-            probationPractitioner: {
-              name: 'Sherlock Holmes',
-              staffIdentifier: 3000,
-            },
-            insidePilot: true,
-          },
-        ],
-        statusConfig,
-        teamView: true,
-        search: 'rogan',
-      })
-      expect(caseloadService.getTeamCreateCaseload).toHaveBeenCalledWith(res.locals.user)
-      expect(caseloadService.getStaffCreateCaseload).not.toHaveBeenCalled()
-    })
-
-    it('should identify offenders outside the pilot areas for the single officer view', async () => {
-      config.rollout.restricted = true
-      config.rollout.prisons = ['LEI']
```

Comment:
Removing the tests which check for cases outside the pilot areas - these checks are already done in the caseload service, so now trusting the status returned for NOT_IN_PILOT - they could be either location restrictions, or because they are recalls or BOTUS type cases.